asset-monitor/src/agent.spec.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-shadow */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
These ESLint checks should not be disabled. Each of the issues they pertain to seems valid.

I will make the changes and push.